### PR TITLE
fetching DappMaintenance contract data

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

**Comment by clementbalestrat** [comment]
@bachstatter what do you think about the logic implemented here to check from the `DappMaintenance` contract if the app should be up or down?

File: netlify.toml [hunk 1]
```diff
@@ -7,11 +7,15 @@
   NODE_OPTIONS = "--max_old_space_size=4096"
   NODE_VERSION = "10.16.3"
   RUBY_VERSION = "2.6.2"
+  REACT_APP_CONTEXT = "production"
```
Comment (clementbalestrat) [inline netlify.toml]: @bachstatter I only want target the dApp in production, not staging.

Comment (bachstatter) [reply netlify.toml]: Yea I think that make sense!

**Comment by bachstatter** [comment]
Looks good. You dont have to keep the interval ID in state. You could just store it in a `const` in the useEffect

**Comment by bachstatter** [comment]
https://github.com/Synthetixio/synthetix-mintr/blob/staging/src/screens/Dashboard/Dashboard.js#L204

File: src/pages/Root/Root.js [hunk 2]
```diff
@@ -46,14 +45,14 @@ const Root = () => {
 	}, []);
 	useEffect(() => {
 		if (process.env.REACT_APP_CONTEXT !== 'production') return;
+		let intervalId;
```
Comment (clementbalestrat) [inline src/pages/Root/Root.js]: @bachstatter had to use `let` as I create the interval inside the `init()` function

Comment (bachstatter) [reply src/pages/Root/Root.js]: Looks good

**Pull request merged**